actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -14,10 +14,17 @@ module MimeNegotiation
       #
       # For backward compatibility, the post \format is extracted from the
       # X-Post-Data-Format HTTP header if present.
-      def content_mime_type
+      def content_mime_type(validate: true)
```

Comment:
@rafaelfranca seems to me the changeset here could be something like

```ruby
- def content_mime_type
+ def content_mime_type(raise_errors_on_unsupported_media_type: Rails.configuration.raise_errors_on_unsupported_media_type)
```

where `Rails.configuration.raise_errors_on_unsupported_media_type` defaults to `false`

Reply:
Yes, something similar to that. But we can't make Action Pack depend on Rails so we need to inverse this dependency. The Railtie configure Action Pack but the config is on this module.